dev/tests/integration/framework/Magento/TestFramework/ErrorLog/Logger.php

```diff
@@ -27,9 +31,14 @@ class Logger extends \Magento\Framework\Logger\Monolog
      * @param HandlerInterface[] $handlers   Optional stack of handlers, the first one in the array is called first, etc
      * @param callable[]         $processors Optional array of processors
      */
-    public function __construct($name, array $handlers = [], array $processors = [])
-    {
-        $this->minimumErrorLevel = defined('TESTS_ERROR_LOG_LISTENER_LEVEL') ? TESTS_ERROR_LOG_LISTENER_LEVEL : -1;
+    public function __construct(
+        $name,
+        array $handlers = [],
+        array $processors = []
+    ) {
```

Comment:
```suggestion
    public function __construct($name, array $handlers = [], array $processors = [] )
    {
```

Reply:
Hello @ihor-sviziev 
Thank you for your suggestion.
I strive for perfection and try to write my code according to PSR-12 (p.2.3).
This line contains more than 80 symbols.